src/backend/renderer/multigpu/mod.rs

```diff
@@ -1916,3 +1916,45 @@ where
             .map_err(Error::Render)
     }
 }
+
+impl<'a, 'b, R: GraphicsApi, T: GraphicsApi, Target, B1, B2> Blit<B1, B2>
```

Comment:
I assume blits cannot cross the device boundary.

Reply:
Blits happen between frame buffers, so they always affect the "target" device in the multigpu context and through the `Bind`-requirement (that also only affects the target), it is not possible to copy from frame buffers of another device right now.

Theoretically, we could support the api for arbitrary frame buffers, by requiring (`ExportDma` and `ImportDma`) or (`ExportMem` and `ImportMem`) instead, but I don't see a way to provide this without making the case this is supposed to optimize slower without specialization.
Given downstream can already manually use `Export/Import[Dma/Mem]` for this use-case, I opted to only implement the direct frame buffer to frame buffer copy.